framework/aster-frame/src/task/processor.rs

```diff
@@ -97,12 +97,9 @@ fn switch_to_task(next_task: Arc<Task>) {
     let current_task_cx_ptr = match current_task() {
         None => PROCESSOR.lock().get_idle_task_cx_ptr(),
         Some(current_task) => {
-            let mut task = current_task.inner_exclusive_access();
+            let cx_ptr = current_task.ctx().get();
```

Comment:
Could you do me a favor by replace all `cx_ptr` with `ctx_ptr` in a separate commit? "Ctx" should be used as the abbreviation for context consistently.